website/.jenkins/append_index_html_to_internal_links.py

```diff
@@ -29,12 +29,18 @@
   'sudo apt-get install python-beautifulsoup4'.
 
 """
+from __future__ import print_function
```

Comment:
The website source code is currently being migrated from https://github.com/apache/beam-site, but is not yet ready. Website changes in apache/beam will be overwritten on next merge. Please contribute changes at apache/beam-site according to the [website contribution guide](https://beam.apache.org/contribute/#contributing-to-the-website). You can track migration progress via [[BEAM-4493]](https://issues.apache.org/jira/browse/BEAM-4493).

Do you know if this change was also made in the apache/beam-site repository? If not, it will need to be migrated.